### PR TITLE
Changed Ruby to 2.4.0

### PR DESCRIPTION
**Pull request by Rdore88**

### CODE REVIEW

**Comment by jenniferoakes** [comment]
@rposborne may I merge this?

**Comment by rposborne** [comment]
@Rdore88 why are you rolling back to 2.4.1?   You are going to require your entire team to install a new ruby by this change.

**Comment by Rdore88** [comment]
@rposborne Sorry Patrick and I were playing around with it for a little bit. I changed it back but I keep getting this message:
/home/travis/build/CareerAtlas/career_atlas/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/connection_adapters/postgresql_adapter.rb:675:in `rescue in connect': FATAL:  database "CareerAtlas_test" does not exist (ActiveRecord::NoDatabaseError)

**Comment by Rdore88** [comment]
@jenniferoakes @rposborne I got it fixed but I can't tell if the front end tests are passing.

**Comment by jenniferoakes** [comment]
@Rdore88 @rposborne wait until tomorrow until we can review in person?

File: config/database.yml [hunk 1]
```diff
@@ -56,9 +56,9 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
-  database: CareerAtlas_test
-
+  adapter: postgresql
+  database: travis_ci_test
```
Comment (rposborne) [inline config/database.yml]: Doesn't this break your tests locally? you changed your db name?   Can't we just ask travis to make a database with the same name?     Inside of our `.travis.yml` we need to run this `psql -c 'create database CareerAtlas_test;' -U postgres` as a before script.   That will ensure that the db exists on travis prior to tests running.

**Review by rposborne — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 1 above)
Comment (Rdore88) [reply config/database.yml]: @rposborne I did what you said and now the local tests work but travis ci doesn't.

Comment (Rdore88) [reply config/database.yml]: @rposborne Your right they did break my local tests but when I changed it travisCI broke.

**Review by rposborne — APPROVED** [review]

**Pull request merged**